src/AppInstallerCLICore/Command.cpp

```diff
@@ -279,10 +245,10 @@ namespace AppInstaller::CLI
         }
 
         // Finally, the link to the documentation pages
-        std::string helpLink = HelpLink();
+        auto helpLink = Utility::LocIndString{ HelpLink() };

```

Comment:
Nit: Doesn't seem too hard to change this one to be loc ind to begin with

Reply:
Correct, this one is simple however it does require changes in around 30+ files. I am trying to avoid making too many changes in a single PR to help maintain a controllable scope. I have this suggestion tracked separately as a follow up 😄